backend/app/transcendence/.eslintrc.js

```diff
@@ -0,0 +1,327 @@
+        "no-multi-assign": "error",
+        "no-nested-ternary": "error",
+        "no-new": "error",
+        "no-var": "error",
+        "spaced-comment":
+        [
+            "error",
+            "always"
+        ],
+        "vars-on-top": "error",
+        "yoda":
+        [
+            "error",
+            "never",
+            {
+                "onlyEquality": true
+            }
+        ],
+        // Layout and Formatting
+        "array-element-newline":
+        [
+            "error",
+            "always"
+        ],
+        "array-bracket-newline":
+        [
+            "error",
+            {
+                "multiline" : true
+            }
+        ],
+        "arrow-parens":
+        [
+            "error",
+            "always"
+        ],
+        "block-spacing": "error",
+        "brace-style":
+        [
+            "error",
+            "allman"
+        ],
+        // "comma-dangle":
+        // [
+        //     "error",
+        //     "always"
+        // ],
+        "comma-spacing":
+        [
+            "error",
+            {
+                "before": false,
+                "after": true
+            }
+        ],
+        "dot-location":
+        [
+            "error",
+            "property"
+        ],
+        "eol-last":
+        [
+            "error",
+            "always"
+        ],
+        "func-call-spacing":
+        [
+            "error",
+            "never"
+        ],
+        "key-spacing":
+        [
+            "error",
+            {
+                "mode": "strict"
+            }
+        ],
+        "line-comment-position":
+        [
+            "error",
+            {
+                "position": "above"
+            }
+        ],
+        "lines-around-comment":
+        [
+            "error",
+            {
+                "beforeBlockComment": true,
+                "allowBlockStart": true,
+                "allowClassStart": true,
+                "allowObjectStart": true,
+            }
+        ],
+        "max-len":
+        [
+            "error",
+            {
+                "code": 80,
+                "tabWidth": 4,
+            }
+        ],
+        "new-parens": "error",
+        "newline-per-chained-call":
+        [
+            "error",
+            {
+                "ignoreChainWithDepth": 2
+            }
+        ],
+        "no-multi-spaces": "error",
+        "no-multiple-empty-lines":
+        [
+            "error",
+            {
+                "max": 2,
+                "maxEOF": 0
+            }
+        ],
+        "no-trailing-spaces":
+        [
+            "error",
+            {
+                "ignoreComments": true
+            }
+        ],
+        "no-whitespace-before-property": "error",
+        "nonblock-statement-body-position":
+        [
+            "error",
+            "below"
+        ],
+        // "object-curly-newline":
+        // [
+        //     "error",
+        //     "always"
+        // ],
```

Comment:
Is this comment needed?

Reply:
Was needed for testing the linter, today it is no longer needed